### DISCUSSION TITLE
Scrollable table body - Vertical

### DISCUSSION THREAD

**Discussion by eafarooqi** [rappasoft/laravel-livewire-tables#867] [Q&A]
Hi, 

how can we make only the body of the table to be scrollable. Right now when we increase the records per page whole page length increases. it will be better if we can make only the tbody scrollable. 

just like this
https://datatables.net/examples/basic_init/scroll_y.html

**Comment by ilsyaa** [comment]
same i also need this

**Reply by ilsyaa** [reply]
@lrljoe

**Reply by lrljoe** [reply]
Should be possible using the setTableAttributes or similar

**Reply by lrljoe** [reply]
@ilsyaa  - Something like this would do it:

```php
public function configure(): void
{
  $this->setTableWrapperAttributes([
    'class' => 'max-h-56 md:max-h-72 lg:max-h-126 overflow-y-scroll',
  ])
  ->setTheadAttributes([
    'class' => 'sticky top-0 '
  ]);
}
```
